sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalogSuite.scala

```diff
@@ -425,4 +425,20 @@ class JDBCTableCatalogSuite extends QueryTest with SharedSparkSession {
       assert(m.contains("\"TABLEENGINENAME\" not found"))
     }
   }
+
+  test("JDBC push down with delimited special identifiers") {
```

Comment:
We actually have a `JDBCV2Suite`. It's probably better to put this test there.

Reply:
Test moved, thanks!